pages/docs/overview/start.md

```diff
@@ -9,19 +9,42 @@ toc: false
 ## Installation Quick Start
 
 ```bash
-git clone {{ site.repo }}.git
-cd singularity
-./autogen.sh
-./configure --prefix=/usr/local
-make
-sudo make install
+$ git clone {{ site.repo }}.git
+$ cd singularity
+$ ./autogen.sh
+$ ./configure --prefix=/usr/local
+$ make
+$ sudo make install
 ```
 
 ## Command Quick Start
 
+### Create a Centos7 image from a CentOS host
+
+It's easiest to build an image on a "compatible" host.  Here's a quick
+recipe to build a CentOS 7 image on a CentOS host.  See
+the [bootstrapping section][readme-bootstrapping] of the README for
```

Comment:
you'll have to put the full URL to the README here, this will likely lead to a 404.